index.html

```diff
@@ -7,7 +7,7 @@
     <title>Project Two</title>
     <link rel="preconnect" href="https://fonts.gstatic.com">
     <link href="https://fonts.googleapis.com/css2?family=Archivo+Black&family=Roboto:wght@100&display=swap" rel="stylesheet"> 
-    <link rel="stylesheet" href="styles/style.css">
+    <link rel="stylesheet" href="styles/sass/style.css">
```

Comment:
would this still be styles/style.css? I thought the style.css file wasn't in the sass folder

Reply:
oh you're right! i need to change the folder